packaging/installer/netdata-support-bundle.ps1

```
@@ -267,25 +329,62 @@ function Invoke-SanitizeLine([string]$line, [bool]$secretScan = $true) {
     return $line
 }
 
-function Test-FileHasNul([string]$path) {
-    # a NUL in the first 1 MiB means binary/UTF-16: line-based redaction would
-    # be byte-unsafe, so the caller withholds the file
-    $fs = [System.IO.File]::OpenRead($path)
+function Get-FileEncodingFacts([string]$path) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-FileEncodingFacts' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-FileEncodingFacts'

(PSUseSingularNouns)